docs/user/index.rst

```diff
@@ -5,6 +5,7 @@ User Guide
    :maxdepth: 1
 
    install
+   upgrade
```

Comment:
can we make this the last section

Reply:
IMO it doesn't matter right now, it'll need restructuring regardless. But I'll move it to the end then.